source/tutorials/source-connector.txt

```diff
@@ -0,0 +1,307 @@
+Tutorial Setup (TODO: link).
+
+.. procedure::
+   :style: connected
+
+   .. step:: Configure the Source Connector
+
+      Create an interactive shell session on the tutorial Docker Container
+      using the following command:
+
+      .. code-block:: bash
+
+         docker run --rm --name tutorialshell --network mongodb-kafka-base_localnet -it robwma/mongokafkatutorial:latest bash
+
+      Create a source configuration file called ``simplesource.json`` with
+      the following command:
+
+      .. code-block:: bash
+
+         nano simplesource.json
+
+      Paste the following configuration information into the file and save
+      your changes:
+
+      .. code-block:: json
+
+         { "name": "mongo-simple-source",
+           "config": {
+             "connector.class": "com.mongodb.kafka.connect.MongoSourceConnector",
+             "connection.uri": "mongodb://mongo1",
+             "database": "Tutorial1",
+             "collection": "orders"
+           }
+         }
+
+      Run the following command in the shell to start the source connector
+      using the configuration file you created:
+
+      .. code-block:: bash
+
+         cx simplesource.json
+
+      .. note::
+
+         The ``cx`` command is a custom script included in the tutorial
+         development environment. This script runs the following
+         equivalent request to the Kafka Connect REST API to create a new
+         connector:
+
+         .. code-block:: bash
+
+            curl -X POST -H "Content-Type: application/json" -d @simplesource.json http://connect:8083/connectors -w "\n"
+
+      Run the following command in the shell to check the status of the
+      connectors:
+
+      .. code-block:: bash
+
+         status
+
+      If your source connector started successfully, you should see the
+      following output:
+
+      .. code-block:: none
+         :copyable: false
+
+         Kafka topics:
+         ...
+
+         The status of the connectors:
+
+         source  |  mongo-simple-source  |  RUNNING  |  RUNNING  |  com.mongodb.kafka.connect.MongoSourceConnector
+
+         Currently configured connectors
+
+         [
+           "mongo-simple-source"
+         ]
+         ...
+
+   .. step:: Create Change Events
+
+      In the same shell, connect to MongoDB using ``mongosh``, the MongoDB
+      shell by running the following command:
+
+      .. code-block:: shell
+
+         mongosh "mongodb://mongo1"
+
+      After you connect successfully, you should see the following
+      MongoDB shell prompt:
+
+      .. code-block::
+         :copyable: false
+
+         rs0 [direct: primary] test>
+
+       At the prompt, type the following commands to insert a new document:
+
+      .. code-block:: javascript
+
+         use Tutorial1
+         db.orders.insertOne( { 'order_id' : 1, 'item' : 'coffee' } )
+
+      Once MongoDB completes the insert command, you should receive an
+      acknowledgment that resembles the following text:
```

Comment:
issue: Should we say "MongoDB" here and personify the database?

```suggestion
      Once the insert command completes, you should receive an
      acknowledgment that resembles the following text:
```

Reply:
I think the style guide mentions that it's ok if it's normal usage of a term.

This version keeps it in active voice rather than passive voice.